pom.xml

```diff
@@ -342,21 +338,66 @@
             </goals>
             <configuration>
               <classifier>${javacpp.platform}${javacpp.platform.extension}</classifier>
-              <skipIfEmpty>true</skipIfEmpty>
-              <includes>
-                <include>${javacpp.platform.library.path}/</include>
-                <include>${javacpp.platform}${javacpp.platform.extension}/</include>
-                <include>org/bytedeco/javacpp/${javacpp.platform}${javacpp.platform.extension}/</include>
-              </includes>
+              <classesDirectory>${project.build.directory}/native</classesDirectory>
+              <skipIfEmpty>false</skipIfEmpty>
+              <archive>
+                <manifestEntries>
+                  <MultiReleaseOutput>true</MultiReleaseOutput>
+                </manifestEntries>
+              </archive>
               <excludes>
-                <exclude>org/bytedeco/javacpp/windows-*/*.exp</exclude>
-                <exclude>org/bytedeco/javacpp/windows-*/*.lib</exclude>
-                <exclude>org/bytedeco/javacpp/windows-*/*.obj</exclude>
+                <exclude>/windows-*/*.exp</exclude>
```

Comment:
Same here, are these excludes still required? I think they are, so let's at least revert the changes so they are effective.

Reply:
Ok. Reverted.